lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ target_sources(${PROJECT_NAME}
 if(CMAKE_UFE_V2_FEATURES_AVAILABLE)
     target_sources(${PROJECT_NAME}
         PRIVATE
+            MayaUsdObject3d.cpp
+            MayaUsdObject3dHandler.cpp
```

Comment:
Moved these two classes, but had to keep Maya versions to override new virtual methods.